docs/logging-usage.rst

```diff
@@ -0,0 +1,295 @@
+Using the API
+=============
+
+Authentication / Configuration
+------------------------------
+
+- Use :class:`Client <gcloud.logging.client.Client>` objects to configure
+  your applications.
+
+- :class:`Client <gcloud.logging.client.Client>` objects hold both a ``project``
+  and an authenticated connection to the Logging service.
+
+- The authentication credentials can be implicitly determined from the
+  environment or directly via
+  :meth:`from_service_account_json <gcloud.logging.client.Client.from_service_account_json>`
+  and
+  :meth:`from_service_account_p12 <gcloud.logging.client.Client.from_service_account_p12>`.
+
+- After setting ``GOOGLE_APPLICATION_CREDENTIALS`` and ``GCLOUD_PROJECT``
+  environment variables, create a :class:`Client <gcloud.logging.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import logging
+     >>> client = logging.Client()
+
+Writing log entries
+-------------------
+
+Write a simple text entry to a log.
+
+.. doctest::
+
+   >>> from gcloud import logging
+   >>> client = logging.Client()
+   >>> log = client.log('log_name')
+   >>> log.text("A simple entry")  # API call
+
+Write a dictionary entry to a log.
+
+.. doctest::
+
+   >>> from gcloud import logging
+   >>> client = logging.Client()
+   >>> log = client.log('log_name')
+   >>> log.struct(message="My second entry",
+   ...     weather="partly cloudy")  # API call
+
+Retrieving log entries
+----------------------
+
+Fetch entries for the default project.
+
+.. doctest::
+
+   >>> from gcloud import logging
+   >>> client = logging.Client()
+   >>> entries, token = client.list_entries()  # API call
```

Comment:
I recall participating in this argument a few times before, still hating the fact that we expose a token to people, who then go "WTF? I wanted to just list all the things... why are you giving me a token and making *me* write code? *you* should write it!"

If users want to be explicit about paging -- which they sometimes will -- that should be possible, but I don't love that as the default.

In other words, listing `thing`s:

```python
# Just give me all the things.
for thing in client.list_things(some=extra, params=here):
  print thing.name

# I actually want to paginate through these, because I'm building a web app that displays them:
page_token = request.get('page_token')
things, next_token = client.get_things_page(page_token=page_token, page_size=20, some=extra, params=here)
return response.render_template('my_template.html', context={'things': things})
```

I totally see the value in having a token and paginating through, but I think that doesn't follow our mantra of "make the common-case fast (to write)".